CRMApi/Models/AbstractModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using CRMApi.AWS.DynamoDB.Linq;
+
+namespace CRMApi.Models {
+    public abstract class AbstractModel<T> {
+        public static QueryableServerData<T> Select () {

```

Comment:
Style thing.  A pattern I've observed when having an abstract model is to add an `All` property so you can do `Post.All.Where(p => p)` and have other static properties that are basically prefiltered like `Post.Published.Where(p => p)`.  Also solves for yoda-LINQ.